web/sass-files/sass/partials/_post.scss

```diff
@@ -374,9 +374,9 @@ body.ios {
 	ul {
 		margin: 0;
 		padding: 0;
-		list-style: none;
```

Comment:
@hmhealey When I merged the permalinks stuff I had merge conflicts with some of Assad's changes. This is probably just the result of a bad merge.